src/go/rpk/pkg/cli/cluster/partitions/unsafe_recover.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package partitions
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newUnsafeRecoveryCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var (
+		dry       bool
+		noConfirm bool
+		nodes     []int
+	)
+	cmd := &cobra.Command{
+		Use:   "unsafe-recover",
+		Short: "Recover unsafely from partitions that have lost majority",
+		Long: `Recover unsafely from partitions that have lost majority.
+
+This command allows you to unsafely recover all data adversely affected by the 
+loss of the nodes specified in the '--from-nodes' flag. This operation is unsafe 
+as it allows you to bulk move all partitions that have lost majority when nodes 
+are gone and irrecoverable.
+
+By performing an unsafe recovery, you are implicitly marking the nodes as 
+'defunct', indicating that Redpanda will consider these nodes unavailable. 
+This action is irreversible.
+
+You may perform a dry run and verify the partitions that will be recovered by 
+using the '--dry' flag.
+`,
+		Run: func(cmd *cobra.Command, _ []string) {
+			f := p.Formatter
+			if f.Kind != "text" && !dry {
+				out.Die("Format type %q is only valid for dry runs (--dry)", f.Kind)
+			}
+			if h, ok := f.Help(adminapi.MajorityLost{}); ok {
+				out.Exit(h)
+			}
+			p, err := p.LoadVirtualProfile(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+			out.CheckExitCloudAdmin(p)
+
+			cl, err := adminapi.NewClient(fs, p)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			plan, err := cl.MajorityLost(cmd.Context(), nodes)
+			out.MaybeDie(err, "unable to get recovery plan: %v", err)
+			err = printPlan(f, plan)
+			out.MaybeDieErr(err)
+			if dry {
+				return
+			}
+			if !noConfirm {
+				confirmed, err := out.Confirm("Confirm recovery from these nodes?")
+				out.MaybeDie(err, "unable to confirm recovery plan: %v", err)
+				if !confirmed {
+					out.Exit("Command execution canceled")
+				}
+			}
+			fmt.Println("Executing recovery plan...")
+			err = cl.ForceRecoverFromNode(cmd.Context(), plan, nodes)
+			out.MaybeDie(err, "unable to execute recovery plan: %v", err)
+			fmt.Println("Successfully queued the recovery plan, you may check the status by running 'rpk cluster partitions balancer-status'")
```

Comment:
`move-status`?

Reply:
I didn't see any change in the move-status command, and the Core's PR added new fields in the balancer-status response: https://github.com/redpanda-data/redpanda/pull/13943

Reply:
Ah, I see, thanks!

Reply:
It looks like there are two fields being added per #13943; `partitions_pending_force_recovery_count` and `partitions_pending_force_recovery_sample`, but the `balancer-status` command doesn't care about them, am I correct? We should update https://github.com/redpanda-data/redpanda/blob/dev/src/go/rpk/pkg/cli/cluster/partitions/status.go accordingly, no? @r-vasquez

Reply:
Yes, I added them to the API response in: [src/go/rpk/pkg/adminapi/api_cluster.go L50](https://github.com/redpanda-data/redpanda/pull/15300/files#diff-26a60a6cf5837e07ef747ecbcf45ce93a0f78ee4a9601ff3f61c79da8e4651acR50) and in the rpk response in [src/go/rpk/pkg/cli/cluster/partitions/status.go](https://github.com/redpanda-data/redpanda/pull/15300/files#diff-31c4a76f84b7c4769741e452d7d678de9c61c89204dbfa798a5518128d15eff4R103) 

It looks like (last row):

```
$ rpk cluster partitions balancer-status 
Status:                               ready
Seconds Since Last Tick:              26
Current Reassignment Count:           0
Partitions Pending Recovery (1):      [kafka/bar/0]
```

Reply:
aha, overlooked it. Great stuff!